api/util_test.go

```diff
@@ -53,7 +53,7 @@ func testJobWithScalingPolicy() *Job {
 	job.TaskGroups[0].Scaling = &ScalingPolicy{
 		Policy:  map[string]interface{}{},
 		Min:     int64ToPtr(1),
-		Max:     int64ToPtr(1),
+		Max:     int64ToPtr(5),
```

Comment:
fixing the bug meant that a test which had previously (incorrectly) passed started to fail. this makes room for that test to properly succeed.